invenio_deposit/tasks.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not needed since this already exists: https://github.com/inveniosoftware/invenio-indexer/blob/master/invenio_indexer/tasks.py#L44